Document/0x06b-Basic-Security-Testing.md

```diff
@@ -573,3 +573,26 @@ You can remotely sniff all traffic in real-time on iOS by [creating a Remote Vir
 ```shell
 ip.addr == 192.168.1.1 && http
 ```
+
+### Allow application installation on a non-iPad device
+
+Sometimes an application can require to be used on a iPad device. If you only have iPhone or iPod touch devices then you can force the application to accept to be installed and used on these kinds of devices by changing the value of the property **UIDeviceFamily** to the value **1** in the **Info.plist** file. 
```

Comment:
1. can you please change "on a iPad device" to "on an iPad device"?
2. can you please split sentence into `and used on these kinds of devices. You can do this by changing`

Reply:
For sure.